runtime/oti/VMHelpers.hpp

```diff
@@ -1728,6 +1729,120 @@ class VM_VMHelpers
 		return rc;
 	}
 
+	/**
+	 * Determine if the method name corresponds to a VarHandle method with polymorphic
+	 * signature.
+	 *
+	 * @param methodNameData the bytes of the method name
+	 * @param methodNameLength the length of the method name
+	 *
+	 * @return true for a VarHandle method with polymorphic signature. Otherwise,
+	 * return false.
+	 */
+	static VMINLINE bool
+	isPolymorphicVarHandleMethod(const U_8 *methodNameData, U_32 methodNameLength)
+	{
+		bool result = false;
+
+		switch (methodNameLength) {
+		case 3:
+			if ((0 == memcmp(methodNameData, "get", methodNameLength))
+				|| (0 == memcmp(methodNameData, "set", methodNameLength))
+			) {
```

Comment:
I strongly prefer this formatting but won't make you change it in new code.
```suggestion
			if ((0 == memcmp(methodNameData, "get", methodNameLength))
			|| (0 == memcmp(methodNameData, "set", methodNameLength))
			) {
```